src/base/profile.h

```diff
@@ -83,9 +83,6 @@ class Profile
     static Profile *m_instance;
 };
 
-inline QString specialFolderLocation(SpecialFolder folder)
-{
-    return Profile::instance().location(folder);
-}
+QString specialFolderLocation(SpecialFolder folder);
```

Comment:
AFAIK, it cannot be inlined when you moved its definition in .cpp

Reply:
the linker can still inline it if "link time optimization" is used, we don't inline the more widely used LogMsg(), so this shouldn't be eligible either IMO.